src/main/java/io/jenkins/plugins/designlibrary/Home.java

```diff
@@ -45,4 +47,15 @@ public UISample getDynamic(String name) {
         }
         return null;
     }
+
+    public String getPluginVersion() {
+        Jenkins jenkins = Jenkins.get();
+        if (jenkins != null && jenkins.getPluginManager() != null) {
```

Comment:
While Jenkins can be `null`, the plugin manager can't. 
```suggestion
        if (jenkins != null) {
```

Reply:
I think you meant to remove jenkins!=null and let the other remain the same , right?

Reply:
No, `jenkins` can be `null`. 

`jenkins.getPluginManager()` != `null`

Reply:
Okay , got it